version.cmake

```diff
@@ -18,15 +18,16 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>. 
 
-set(MUSE_APP_NAME "MuseScoreStudio")
-set(MUSE_APP_NAME_WITH_SPACE "MuseScore Studio")
-set(MUSE_APP_GUI_IDENTIFIER org.musescore.MuseScore)
+set(MUSE_APP_NAME "MuseScore")
+set(MUSE_APP_TITLE "MuseScore Studio")
+set(MUSE_APP_GUI_IDENTIFIER org.musescore.${MUSE_APP_NAME})
```

Comment:
This affects the bundle identifier on macOS. I would advise against doing that unless we are _very_ confident about the implications of this.

Reply:
Oh nevermind, I see you changed MUSE_APP_NAME back... it's confusing with so many variables 😄 (but it's fine)

Reply:
It remains as it was before MuseScore, I haven't changed the bundle identifier